internal/controllers/reconciliation/controller.go

```diff
@@ -37,6 +38,9 @@ type Options struct {
 	Timeout               time.Duration
 	ReadinessPollInterval time.Duration
 	MinReconcileInterval  time.Duration
+	
+	// ResourceSelector filters which resources within compositions should be reconciled
```

Comment:
No need to document this - it's obvious and the other fields don't have comments so visually it stands out too much